qemu/tests/x86_cpu_model.py

```diff
@@ -72,6 +72,17 @@ def run(test, params, env):
     cmd = params["get_model_cmd"]
     out = session.cmd_output(cmd)
     if not re.search(guest_model, out):
+        check_cmd = params.get("check_cmd")
+        if check_cmd:
+            vulnerabilities = params.get("check_items").split(" ")
```

Comment:
No `check_items` defined for this case, you want to use `vulnerabilities` in configuration file?

Reply:
> No `check_items` defined for this case, you want to use `vulnerabilities` in configuration file?

Thanks and update.